acceptance/setup/aio/pre-suite/010_Install.rb

```diff
@@ -61,35 +47,20 @@
       master.install_package('puppetserver')
 
       logger.info "EC2 master found: Installing #{ENV['SHA']} build of puppet-agent."
-      # Upgrade installed puppet-agent with targeted SHA.
-      opts = {
-        :puppet_collection => 'PC1',
-        :puppet_agent_sha => ENV['SHA'],
-        :puppet_agent_version => ENV['SUITE_VERSION'] || ENV['SHA'] ,
-        :dev_builds_url => "http://builds.delivery.puppetlabs.net"
```

Comment:
I don't understand how this worked.

Above we have the comment
```
       # An EC2 master instance does not have access to puppetlabs.net for getting
       # dev repos.
       #
       # We will install the appropriate repo to satisfy the puppetserver requirement
       # and then upgrade puppet-agent with the targeted SHA package afterwards.
       #
       # Currently, only an `el` master is supported for this operation.
```

And yet we previously had been installing puppet-agent from `builds.delivery.puppetlabs.net`. My only guess is that we were never actually hitting this code.

My changes don't change this functionality. They also hit `builds.delivery.puppetlabs.net` to install the puppet-agent package.